docs/gen-ai/gen-ai-events.md

```diff
@@ -0,0 +1,380 @@
+
+Body fields that contain user input, model output, or other potentially sensitive and verbose data
+SHOULD NOT be captured by default.
+
+Semantic conventions for individual systems which extend content events SHOULD document all additional body fields and specify whether they
+should be captured by default or need application to opt into capturing them.
+
+Telemetry consumers SHOULD expect to receive unknown body fields.
+
+Instrumentations SHOULD NOT capture undocumented body fields and MUST follow the documented defaults for known fields.
+Instrumentations MAY offer configuration options allowing to disable events or allowing to capture all fields.
+
+## Common attributes
+
+The following attributes apply to all GenAI events.
+
+<!-- semconv gen_ai.common.event.attributes -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| [`gen_ai.system`](/docs/attributes-registry/gen-ai.md) | string | The Generative AI product as identified by the client or server instrumentation. [1] | `openai` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+**[1]:** The `gen_ai.system` describes a family of GenAI models with specific model identified
+by `gen_ai.request.model` and `gen_ai.response.model` attributes.
+
+The actual GenAI product may differ from the one identified by the client.
+For example, when using OpenAI client libraries to communicate with Mistral, the `gen_ai.system`
+is set to `openai` based on the instrumentation's best knowledge.
+
+For custom model, a custom friendly name SHOULD be used.
+If none of these options apply, the `gen_ai.system` SHOULD be set to `_OTHER`.
+
+
+
+`gen_ai.system` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
+
+| Value  | Description | Stability |
+|---|---|---|
+| `anthropic` | Anthropic | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `cohere` | Cohere | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `openai` | OpenAI | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `vertex_ai` | Vertex AI | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+
+
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+<!-- END AUTOGENERATED TEXT -->
+<!-- endsemconv -->
+
+## System event
+
+This event describes the instructions passed to the GenAI model.
+It's NOT RECOMMENDED to report this event when capturing content is disabled, unless there are additional system-specific fields
```

Comment:
Is this specific to the system event or other events? e.g. the user events

Reply:
I think it's fine, thanks for the clarification

Reply:
I removed this sentence to simplify things.